src/components/Hero.tsx

```
@@ -42,20 +42,23 @@ const Hero: React.FC = () => {
             </svg>
             <span>Featured on Hacker News</span>
           </a>
-          <div dangerouslySetInnerHTML={{ __html: productHuntBadgeHtml }} />
+          <div
+            className="scale-90"
+            dangerouslySetInnerHTML={{ __html: productHuntBadgeHtml }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)